wire-grpc-tests/src/test/proto-grpc/routeguide/Feature.kt

```diff
@@ -55,7 +55,7 @@ class Feature(
   override fun equals(other: Any?): Boolean {
     if (other === this) return true
     if (other !is Feature) return false
-    return unknownFields == other.unknownFields
+    return unknownFields() == other.unknownFields()
```

Comment:
no longer a val. I couldn’t figure out how to have a nullable JVM field named `unknownFields` and also a non-nullable val with the same name.

Reply:
You probably need to do something like
```kotlin
@JvmName("unknownFields")
private val _unknownFields: ByteString?

val unknownFields get() = _unknownFields ?: EMPTY
```

Reply:
I tried it and IntelliJ complained. Lemme try again.

Reply:
Yeah, `@JvmName` can’t target field names.

```
/**
 * Specifies the name for the Java class or method which is generated from this element.
 *
 * See the [Kotlin language documentation](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#handling-signature-clashes-with-jvmname)
 * for more information.
 * @property name the name of the element.
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER, AnnotationTarget.FILE)
@Retention(AnnotationRetention.BINARY)
@MustBeDocumented
public actual annotation class JvmName(actual val name: String)
```